### ISSUE TITLE
Partial charge information missing from the state descriptions

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/molecule-polarity#287]
**Test device**
Asus Zenbook

**Operating System**
Win11

**Browser**
Chrome 

**Problem description**
From https://github.com/phetsims/qa/issues/1355

In the A11y view during the Real Molecules screen, the partial charges information displayed for all molecules is missing when the Advanced Model option is selected.

**Visuals**

<img width="1600" height="573" alt="Image" src="https://github.com/user-attachments/assets/eec7e47b-c4ae-435e-a1d8-1555de6ebb1f" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Molecule Polarity‬
URL: https://phet-dev.colorado.edu/html/molecule-polarity/2.0.0-dev.1/phet/molecule-polarity_all_phet.html
Version: 2.0.0-dev.1 2026-02-06 16:31:15 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36
Language: en-US
Window: 1152x585
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>
